### DISCUSSION TITLE
Where are passes stored?

### DISCUSSION THREAD

**Discussion by dandspach** [jekhokie/raspberry-noaa-v2#594] [Q&A] [answered]
I want to query the text of scheduled passes (not on webpage) and scroll them across an led marquee sign.  Does anyone know where they are 'stored'?

**Comment by ysamouhos** [comment, marked as answer]
The info is in sqlite3 db in /home/pi/raspberry-noaa-v2/db/panel.db if you have the default install. 
There are 2 tables "predict_passes" and "decoded_passes". 
Obviously the fist is the passes to come and the second the decoded ones.
